### DISCUSSION TITLE
Fdtransfer, glibc on host, musl on container

### DISCUSSION THREAD

**Discussion by krzysztofslusarski** [async-profiler/async-profiler#722] [Q&A] [answered]
Hi, is it possible to run AP with fdtransfer with:
- glibc on host
- musl on container?
Should I unpack glibc distro on host and musl distro inside container in the same dir?

**Comment by apangin** [comment, marked as answer]
Yes, it's possible. You can even use the same glibc distro both on the host and in the container (or just mount the profiler directory in the container).
You cannot run glibc `jattach` on a musl system, but glibc `libasyncProfiler.so` should work on both.

**Reply by krzysztofslusarski** [reply]
Thank you
